src/OSPSuite.R/Domain/SimulationBatch.cs

```diff
@@ -77,8 +77,10 @@ ISimulationResultsCreator simulationResultsCreator
          _simulationResultsCreator = simulationResultsCreator;
       }
 
-      public void Initialize(IModelCoreSimulation simulation, SimulationBatchOptions simulationBatchOptions)
+      public void Initialize(IModelCoreSimulation simulation, SimulationBatchOptions simulationBatchOptions, SimulationRunOptions simulationRunOptions)
       {
+         _simModelBatch.CheckForNegativeValues = simulationRunOptions?.CheckForNegativeValues ?? true;
```

Comment:
There was a difference in how this parameter was set for batch vs regular simulation runner. In the batch, there was actually no way to manipulate this option. It was hard-coded.
It seems like you should be able to manipulate it.